capitolwords_ng/parsers/text_utils.py

```diff
@@ -26,10 +34,13 @@ def preprocess(text):
     text = re.sub('HON\.', 'HON', text)
```

Comment:
Alternatively, create a top level variable for each with a descriptive name. That might be easier, especially if we roll this into a class it could be extended to be document specific (each type gets its own batch of regexes to apply). Thats probably overengineering for what we're shooting for right now though.